Cargo.toml

```diff
@@ -3,7 +3,7 @@ resolver = "2"
 members = ["crates/*"]
 
 [workspace.package]
-version = "0.6.0"
+version = "0.7.1"
```

Comment:
So we manually increment version immediately after release and crate publish on crates.io?

Reply:
Yes in order to be able to align with current version in place on github